com.unity.shadergraph/Editor/Drawing/Views/GraphSubWindow.cs

```diff
@@ -42,24 +55,34 @@ class GraphSubWindow : GraphElement, ISGResizable
         public virtual string elementName => "";
         public virtual string windowTitle => "";
 
-        public GraphView graphView
+        public VisualElement ParentView
         {
             get
             {
-                if (!isWindowed && m_GraphView == null)
-                    m_GraphView = GetFirstAncestorOfType<GraphView>();
-                return m_GraphView;
+                if (!isWindowed && m_ParentView == null)
+                    m_ParentView = GetFirstAncestorOfType<GraphView>();
+                return m_ParentView;
             }
 
             set
             {
                 if (!isWindowed)
                     return;
-                m_GraphView = value;
+                m_ParentView = value;
             }
         }
 
-        public List<ISelectable> selection => graphView?.selection;
+        public List<ISelectable> selection
+        {
+            get
+            {
+                if (ParentView is ISelectionProvider selectionProvider)
+                    return selectionProvider.GetSelection;
+
+                Debug.Log("ERROR: GraphSubWindow: Was unable to find a selection provider. Please check if parent view of: " + name + " implements ISelectionProvider::GetSelection");
```

Comment:
Not user actionable, should probably be guarded assert.

Reply:
Converted to a guarded assert call